.github/workflows/cleanEnvironments.js

```diff
@@ -47,73 +46,61 @@ program
   })
   .parse();
 
-async function getEnvironmentsToClean(token, environmentsToCleanRegex) {
-  const environments = await getEnvironments(token);
-  const environmentsForReleaseTesting = environments.filter(environment => {
-    return environment.domain.match(environmentsToCleanRegex)
+async function getEnvironmentsToClean(
+  allEnvironments,
+  environmentsToCleanRegex,
+) {
+  const environmentsForReleaseTesting = allEnvironments.filter(environment => {
+    return environment.domain.match(environmentsToCleanRegex);
   });
   return environmentsForReleaseTesting;
 }
 
-async function getEnvironments(token) {
-  const response = await fetch(
-    `${pathToCloudAPI}organizations/saleor/environments/`,
-    {
-      method: "GET",
-      headers: {
-        Authorization: `Token ${token}`,
-        Accept: "application/json",
-        "Content-Type": "application/json;charset=UTF-8",
-      },
-    },
-  );
-  return await response.json();
+async function getEnvironments() {
+  const result = await exec("npx saleor env list --json", {
+    serialization: "json",
+  });
+  const envList = result.stdout.replace("\x1B[?25l\x1B[?25h", "");
```

Comment:
Can you add comment on what you are doing here? (the ascii escape part: `\x1B[?25l\x1B[?25h` - which is probably hiding and showing the cursor)

Does saleor-cli have a pipe mode or CI mode so we don't need to remove the ASCII escapes?